extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -580,7 +580,10 @@ export class ProjectsController {
 			} else if ((<ISystemDatabaseReferenceSettings>settings).systemDb !== undefined) {
 				await project.addSystemDatabaseReference(<ISystemDatabaseReferenceSettings>settings);
 			} else {
-				await project.addDatabaseReference(<IDacpacReferenceSettings>settings);
+				// update dacpacFileLocation to relative path to project file
+				const dacpacRefSettings = settings as IDacpacReferenceSettings;
+				dacpacRefSettings.dacpacFileLocation = vscode.Uri.file(utils.trimUri(vscode.Uri.file(project.projectFilePath), dacpacRefSettings.dacpacFileLocation));
```

Comment:
https://nodejs.org/api/path.html#path_path_relative_from_to

I think would be a better choice than `trimUri()`